### DISCUSSION TITLE
Start Collection

### DISCUSSION THREAD

**Discussion by svpa82** [AntlerVC/firetable#1172] [Support / Q&A] [answered]
Hi, can I use Rowy to edit a collection that is not a Start collection? 
I wan't to add Table and Edit my settings in Rowy that is stored under: 

[Start Collection] > [Document]  > [Start Collection]
settings > settings > companyBranches
settings > settings > companyCategories
etc.

In Rowy I can only find that I can select Start collection to create a Table for.

**Comment by harinij** [comment, marked as answer]
Hi @svpa82 👋 assuming these are sub collections? Currently the drop down list only shows the top levels collections, but you can create a table with for any collection path by simply adding that as the collection name 'settings/settings/companyBranches'. Hope that solves your issue.
